dockerfiles/alpine_jdk/Dockerfile

```diff
@@ -82,6 +82,5 @@ RUN apk upgrade --update && \
             /opt/codenvy-data/che-machines /opt/codenvy-data/che-machines-logs \
             /opt/codenvy-data/conf /opt/codenvy-data/conf/logback \
             /opt/codenvy-data/license
-ADD entrypoint.sh /opt/entrypoint.sh
-RUN chmod a+x /opt/entrypoint.sh
-ENTRYPOINT ["/opt/entrypoint.sh"]
+
+RUN postfix -c /etc/postfix start
```

Comment:
I agree, It was in entrypoint  here but I've moved it to codenvy. 
I don't this we should have this at all here... because if it will be in entrypoint it will be overridded by codenvy entrypoint because codenvy doing FROM this one.
and even if it will work it does not match single process pattern.

So I will remove it from jdk image. We agreed with @TylerJewell that if user wants emails he should configure smtp.